layouts/default.vue

```diff
@@ -3,53 +3,3 @@
     <nuxt/>
```

Comment:
when running via 'npm run dev' this line throws a linting error, it expects an additional space.